docs/tutorials/using-lego-dimensions-toy-pad.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Using the LEGO Dimensions Toy Pad
+subject: Hardware - Other
+author: "@JorgePe"
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+The [LEGO Dimensions game](http://www.lego.com/en-us/dimensions/support) makes
+use of NFC technology to allow some iteraction between physical LEGO creations
+(characters, vehicles and gadgets) and the videogame.
+
+The LEGO Toy Pad is in fact just a custom USB triple NFC reader that can read
+NFC tags (not only the LEGO Toy Tags but also several other tags including those
+used in other games like Disney Infinity) and change the color of the RGB light
+associated to each reader.
+
+## Credits
+
+Since the beginning of 2015 several people have been reverse engineering the
+LEGO Toy Pad.
+This tutorial is strongly based on **woodenphone** [work](https://github.com/woodenphone/lego_dimensions_protocol) available at GitHub.
```

Comment:
GitHub @ mentions work on our site, so you can write `@woodenphone` instead of `**woodenphone**` and it will actually link to his/her GitHub profile.